docs/releases/5.2.txt

```diff
@@ -229,6 +229,11 @@ Management Commands
   ``runserver`` is unsuitable for production. This warning can be hidden by
   setting the :envvar:`HIDE_PRODUCTION_WARNING` environment variable to
   ``"true"``.
+  
+* The :djadmin:`shell` management command now automatically imports all models
+  from all installed apps. :ref:`Customize this behavior
+  <customizing-shell-auto-imports>` to add extra imports or remove the new
+  defaults.
```

Comment:
@nessita how do you feel about this being a headline feature?

Something like:
```
The ``shell`` command automatically imports models
--------------------------------------------------

The :djadmin:`shell` management command now automatically imports all models
from all installed apps.

You can view all imported functions and classes when ``--verbosity`` is 2
or above.

.. code-block:: shell

    $ python manage.py shell --verbosity=2
    10 objects imported automatically
    from django.contrib.sessions.models import Session
    from django.contrib.contenttypes.models import ContentType
    from django.contrib.auth.models import User, Group, Permission
    from django.contrib.admin.models import LogEntry
    import django.contrib.admin.models as admin_models
    import django.contrib.auth.models as auth_models
    import django.contrib.contenttypes.models as contenttypes_models
    import django.contrib.sessions.models as sessions_models

This :ref:`behavior can be customized <customizing-shell-auto-imports>` to add
extra imports or remove the new default imports.
```

Reply:
> @nessita how do you feel about this being a headline feature?

Yes, +1! Sounds like a great idea (and very well deserved).

> Something like:
> 
> ```
> The ``shell`` command automatically imports models
> --------------------------------------------------

I think these headlines are usually a "thing" (noun) and not use a verb. How about:

`Auto imported modules in the ``shell```
 
> The :djadmin:`shell` management command now automatically imports all models
> from all installed apps.
> 
> You can view all imported functions and classes when ``--verbosity`` is 2
> or above.
> 
> .. code-block:: shell
> 
>     $ python manage.py shell --verbosity=2
>     10 objects imported automatically
>     from django.contrib.sessions.models import Session
>     from django.contrib.contenttypes.models import ContentType
>     from django.contrib.auth.models import User, Group, Permission
>     from django.contrib.admin.models import LogEntry
>     import django.contrib.admin.models as admin_models
>     import django.contrib.auth.models as auth_models
>     import django.contrib.contenttypes.models as contenttypes_models
>     import django.contrib.sessions.models as sessions_models

If you could wait for my review, I'd like to do a complete one before the week ends.